OrderList.js

```diff
@@ -9,18 +9,23 @@ export class OrderList {
   }
 
   async fetchData() {
-    const responsesArray = await Promise.all([
-      fetch(
-        "https://my-json-server.typicode.com/Solnick/fake-orders-db/buyers",
-      ),
-      fetch(
-        "https://my-json-server.typicode.com/Solnick/fake-orders-db/orders",
-      ),
-      fetch(
-        "https://my-json-server.typicode.com/Solnick/fake-orders-db/products",
-      ),
-    ]);
-    this.fetchedData = await Promise.all(responsesArray.map((res) => res.json()));
+    const buyersResponse = await fetch(
```

Comment:
Now you the fetching is done one after another. The previous code was more performant. Please get back to the previous code. And just use the array returned from the second `Promise.all` to store proper elements in the class properties.